### DISCUSSION TITLE
Cool network plot

### DISCUSSION THREAD

**Discussion by pz-max** [PyPSA/PyPSA#400] [Show and tell]
Hey guys,
Maybe one of the first discussion posts. Thanks for PyPSA <3
Below image created in this [notebook](https://github.com/pypsa-meets-africa/pypsa-africa/blob/main/notebooks/validation/network_validation.ipynb). We are going to optimize Africa & hope some people will join us to do this on other continents and improve macro-energy system planning around the world.

![image](https://user-images.githubusercontent.com/61968949/168248067-f69f8d74-c8fc-4be5-902b-d25298a0d0b3.png)

**Comment by mdzzg** [comment]
Impressive.
